src/AutoRest.CSharp/MgmtTest/Generation/MgmtBaseTestWriter.cs

```diff
@@ -492,13 +508,13 @@ public void WriteFrameworkTypeExampleValue(CodeWriter writer, CSharpType cst, Ex
             }
         }
 
-        public static void WriteEnumTypeExampleValue(CodeWriter writer, EnumType enumType, ExampleValue exampleValue)
+        public void WriteEnumTypeExampleValue(CodeWriter writer, EnumType enumType, ExampleValue exampleValue)
         {
             if (enumType.IsExtendable)
             {
-                if (new string[] { "String" }.Contains(enumType.BaseType.FrameworkType.Name))
+                if (new string[] { "String" }.Contains(enumType.BaseType.FrameworkType.Name) && exampleValue.RawValue is string strValue)
```

Comment:
Why is this string's first letter in upper case?

Reply:
Because Type.Name is using `System.String`?
I'm changing to below to avoid use Type.Name:
~~~
enumType.BaseType.FrameworkType == typeof(String)
~~~